ribbon/src/test/java/feign/ribbon/RibbonClientTest.java

```diff
@@ -135,6 +138,15 @@ public void ioExceptionRetryWithBuilder() throws IOException, InterruptedExcepti
     // TODO: verify ribbon stats match
     // assertEquals(target.lb().getLoadBalancerStats().getSingleServerStat())
   }
+  
+  @Test
+  public void testFeignOptionsClientConfig() {
+    Request.Options options = new Request.Options(1111,22222);
+    IClientConfig config = new RibbonClient.FeignOptionsClientConfig(options);
+    assertEquals(new Integer(1111),config.get(CommonClientConfigKey.ConnectTimeout));
```

Comment:
one suggestion to make this more clear is only to test the things you are adding.

ex.

```java
assertThat(config.get(CommonClientConfigKey.ConnectTimeout))
  .isEqualTo(options.connectTimeout());
// repeat for read timeout
```